src/librbd/api/Group.cc

```diff
@@ -1226,24 +1216,118 @@ int Group<I>::snap_rename(librados::IoCtx& group_ioctx, const char *group_name,
   return 0;
 }
 
+int convert_cls_rbd_GroupSnapshot_to_librbd_group_snap_info2_t(
```

Comment:
Suggestions welcome for a less kludgy function name

Reply:
I'd go with `GroupSnapshot_to_group_snap_info2()`.